src/Service/OrderHelper.php

```diff
@@ -0,0 +1,61 @@
+<?php declare(strict_types=1);
```

Comment:
interesting that you declare strict, but did not do it on other files :P, also use a new line

Reply:
Here I copy SW6 service structure, will add strict to other files.

No new line in any of SW6 files and example plugins, looks like internal formating.